jenkins/ec2_bootstrap.sh

```diff
@@ -423,6 +423,24 @@ prepare_cowbuilder() {
     /bin/bash ./bootstrap.sh
 }
 
+install_datadog_agent() {
+    # Install the Datadog agent. The agent requires an API key which is expected
+    # to be present in the shell environment as DATADOG_AGENT_API_KEY. If the
+    # value is not present then inform stdout but do not sys exit.
+    if ! [[ -z "${DATADOG_AGENT_API_KEY}" ]]; then
+	echo "Optional envvar DATADOG_AGENT_API_KEY does not exist."
+    else
+	DD_CURL_TO_BASH_URI="https://raw.githubusercontent.com/DataDog/dd-agent/master/packaging/datadog-agent/source/install_agent.sh"
```

Comment:
I'd call this DD_AGENT_INSTALLER_URI. I'd break it into two pieces so the line doesn't go beyond 100 characters..

DD_AGENT_REPO=https://raw.githubusercontent.com/DataDog/dd-agent
DD_AGENT_INSTALLER_URI=$DD_AGENT_REPO/master/packaging/datadog-agent/source/install_agent.sh